apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -119,6 +119,11 @@ public override string ConnectionString
         [Category("Data")]
         public string SpannerInstance => _connectionStringBuilder?.SpannerInstance;
 
+        /// <summary>
+        /// This property is intended for internal use only.
+        /// </summary>
+        public Logger Logger { get; set; } = Logger.DefaultLogger;
```

Comment:
Just to check the intended semantics here - is it okay that changing the default logger after creating an instance of SpannerConnection doesn't change where that connection logs to?

Reply:
Yes, I didn't want the ctor to require a Logger (or even hint at one) since it will not normally be set by the user.  Its expected that if its necessary, this will be overwritten in the object initializer.